crates/rattler_installs_packages/src/resolve.rs

```diff
@@ -1,3 +1,11 @@
+//! This module contains the [`PypiDependencyProvider`] which is used
```

Comment:
this is out of date now. There is just the resolve function and the feature is called resolvo.

Reply:
Done